docs/user-documentation/soc-and-feeds.md

```diff
@@ -113,7 +113,36 @@ In Swarm, `topic` is a 32-byte long arbitrary byte array. It's possible to choos
 const topic = bee.makeFeedTopic('my-dapp.eth/outbox')
 ```
 
-### Reading feeds
+### High level JSON API
+
+High level JSON API is a convenience API that is bound to JSON data types. It allows easily setting and getting of a value to/from feed.
```

Comment:
Maybe an introduction about the why would be nice, something along the lines of:

"Many applications are storing or manipulating data in JSON so bee-js has convenience functionality to use feeds with JSON objects.